includes/admin.form.inc

```diff
@@ -39,13 +39,74 @@ function islandora_newspaper_admin_settings_form(array $form, array &$form_state
     ),
   );
 
-  $form['islandora_newspaper_parent_issue_solr_field'] = array(
-    '#type' => 'textfield',
-    '#title' => t('Parent Solr Field'),
-    '#description' => t("Solr field containing the parent issue's PID."),
-    '#default_value' => variable_get('islandora_newspaper_parent_issue_solr_field', 'RELS_EXT_isMemberOf_uri_ms'),
-    '#size' => 30,
+  $form['islandora_newspaper_solr_options'] = array(
+    '#type' => 'fieldset',
+    '#title' => t('Solr settings'),
+    'islandora_newspaper_parent_issue_solr_field' => array(
+      '#type' => 'textfield',
+      '#title' => t('Parent Solr Field'),
+      '#description' => t("Solr field containing the parent issue's PID."),
+      '#default_value' => variable_get('islandora_newspaper_parent_issue_solr_field', 'RELS_EXT_isMemberOf_uri_ms'),
+      '#size' => 100,
+      '#autocomplete_path' => 'islandora_solr/autocomplete_luke',
+    ),
+    'islandora_newspaper_use_solr' => array(
+      '#type' => 'checkbox',
+      '#title' => t('Use Solr for Newspaper display'),
+      '#disabled' => (!module_exists('islandora_solr')),
+      '#description' => t('Use Solr to generate lists of issues for a newspaper object.'),
+      '#default_value' => (module_exists('islandora_solr') ? variable_get('islandora_newspaper_use_solr', FALSE) :
+                            FALSE),
+    ),
+    'islandora_newspaper_solr_wrapper' => array(
+      '#type' => 'container',
+      '#states' => array(
+        'visible' => array(
+          ':input[name="islandora_newspaper_use_solr"]' => array('checked' => TRUE),
+        ),
+      ),
+      'islandora_newspaper_solr_newspaper_parent_field' => array(
+        '#type' => 'textfield',
+        '#title' => t('Newspaper issue parent field'),
+        '#description' => t('Solr field that contains the parent newspaper of the newspaper issue.'),
+        '#size' => 100,
+        '#default_value' => variable_get('islandora_newspaper_solr_newspaper_parent_field', 'RELS_EXT_isMemberOf_uri_ms'),
+        '#autocomplete_path' => 'islandora_solr/autocomplete_luke',
+        '#states' => array(
+          'required' => array(
+            ':input[name="islandora_newspaper_use_solr"]' => array('checked' => TRUE),
+          ),
+        ),
+      ),
+      'islandora_newspaper_solr_date_field' => array(
+        '#type' => 'textfield',
+        '#title' => t('Issued date field'),
+        '#description' => t('Solr field that contains the date issued of the newspaper issue.'),
+        '#size' => 100,
+        '#default_value' => variable_get('islandora_newspaper_solr_date_field', 'RELS_EXT_dateIssued_literal_ms'),
```

Comment:
No, this does not do that. It would require more work (which I considered) to check if the field was a date and then sort in Solr. But this returns just returns the results which are grouped and sorted in PHP.

Reply:
@whikloj don't want to be insistent, but in the rare case that you actually have an _ms field with multiple values, will the display/after logic )(not yours, what is already in place) work out correctly? What happens if you get two dates? This is just plain curiosity now

Reply:
Ok, so for multi-valued fields I am just [grabbing the first](https://github.com/Islandora/islandora_solution_pack_newspaper/pull/158/files#diff-2d69368f59ba81d5635cd8274dd66090R242). With a date issued it seems like you shouldn't have a bunch of them. But I'm open to suggestions.